README.md

```diff
@@ -335,3 +354,27 @@ constructor(props) {
   this.X = limit(dragX, panState, -100, 100);
 }
 ```
+
+### spring
```

Comment:
I think it should be called `snap` as I'm missing `spring` export

Reply:
thks!